Containerfile

```
@@ -38,8 +38,12 @@ RUN rm /usr/share/nginx/html/index.html
 COPY --from=fedora-build pulp-fixtures/fixtures /usr/share/nginx/html
 COPY --from=debian-build pulp-fixtures/fixtures /usr/share/nginx/html
 
-# turn on autoindex
-RUN sed -i -e '/location \/ {/a autoindex on\;' /etc/nginx/conf.d/default.conf
+# tune nginx config file
+RUN mkdir -p /pulp-fixtures/common
+ADD common /pulp-fixtures/common
+RUN echo "TED SE TO DEJE!"
+RUN rm -f /etc/nginx/nginx.conf
+RUN cp /pulp-fixtures/common/nginx.conf /etc/nginx/nginx.conf
```

Would that not be enough?

```suggestion
COPY common/nginx.conf /etc/nginx/nginx.conf
```